tests/test_cliCmdQueryDatasets.py

```diff
@@ -177,7 +165,7 @@ def testShowURI(self):
                        "visit", "URI")),
         )
 
-        self._assertTablesEqual(tables, expectedTables)
+        assertAstropyTablesEqual(self, tables, expectedTables)
```

Comment:
You probably want assertAstropyTablesEqual to be in a class and then use multiple inheritance in this TestCase so that it inherits from unittest.TestCase and the butlerTestHelper (or whatever you call it). Then you'd still be able to do `self.assertTablesEqual` here and make it look like a normal test assert. We use the multiple inheritance technique a lot in obs_base butler tests.